xapi-xenopsd-xc.opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "1.2"
+name: "xapi-xenopsd-xc"
+maintainer: "xen-api@lists.xen.org"
+authors: "xen-api@lists.xen.org"
+homepage: "https://github.com/xapi-project/xenopsd"
+dev-repo: "https://github.com/xapi-project/xenopsd.git"
+bug-reports: "https://github.com/xapi-project/xenopsd/issues"
+build: [[ "jbuilder" "build" "-p" name "-j" jobs ]]
+build-doc: [[ "jbuilder" "build" "@doc" ]]
+depends: [
+  "jbuilder" {build}
+  "xapi-xenopsd"
+  "cohttp"
+  "core"
+  "ezxenstore"
+  "fd-send-recv"
+  "mtime"
+  "qmp"
+  "re"
+  "rpc"
+  "sexplib"
+  "uri"
+  "uuidm"
+  "uutf"
+  "xapi-forkexecd"
+  "xapi-idl"
+  "xapi-inventory"
+  "xapi-stdext-unix"
+  "xenctrl"
+  "xenstore"
+  "xenstore_transport"
+  "xmlm"
+  "odoc" {doc}
```

Comment:
We've discussed that we've remove these for now, along with the `build-doc` fields. Currently we don't have these in any of our opam files. Once we figure out how to build the docs for a given package using jbuilder, we can re-add this to everything.